networktables.json

```diff
@@ -6,5 +6,21 @@
     "properties": {
       "persistent": true
     }
+  },
+  {
```

Comment:
Did you intend for these to be part of this PR?  They don't seem drive related

Reply:
haha no, stuff I didn't push to armsubsystem